functions.php

```diff
@@ -386,4 +386,100 @@ function ucsc_truss_assets() { ?>
 /**
  * Add Excerpts to Pages
  */
-add_post_type_support( 'page', 'excerpt' );
\ No newline at end of file
+add_post_type_support( 'page', 'excerpt' );
+
+/**
+ * Remove default skip link and add a custom one.
+ */
+remove_action( 'wp_footer', 'the_block_template_skip_link' );
+
+/**
+ * Override the default skip link to pass a11y for landmarks.
+ * @return void
+ */
+function ucsc_skip_link() {
+	/**
+	* Print the skip-link styles.
+	*/
+	?>
+	<style id="skip-link-styles">
```

Comment:
Any chance that we use enqueue styles and script functions here? Instead of just print all items at once?

Reply:
@MlKilderkin what's the advantage there? Typically, with time sensitive scripts and styles we'd do that and have separate files. In this case, we're just duplicating the WP original functionality. see `the_block_template_skip_link()` in the `wp-includes` folder. For that reason, I'm duplicating the original WP functionality and using then loading ours instead.

Reply:
Just want to double check if we could move this somewhere. I am fine with the current solution, thank you